core/build.gradle.kts

```diff
@@ -78,7 +78,7 @@ dependencies {
     testImplementation(kotlin("stdlib-jdk8"))
     testImplementation(kotlin("test"))
     testImplementation(kotlin("test-junit5"))
-    testRuntimeOnly("org.slf4j:slf4j-log4j12")
+    testImplementation("org.slf4j:slf4j-log4j12")
```

Comment:
Why is this change?

Reply:
Oops, i need to reference log4j class in my Test, can we fix that `auto-complete` ?

Reply:
In any case you don't really need `slf4j-log4j12` on the compilation classpath.
You don't really reference slf4j-log4j12 classes.
You reference log4j12 classes.

I see the following options:
a) Keep `slf4j-log4j12` runtime-only and add an extra `testImplementation("log4j:log4j") { because("SqlHintsConverterTest needs to implement a MockAppender") }`
b) Create an extra `src/test-logging` folder that would have extra dependency for testing logging behavior. For instance, `slf4j-test` or `spf4j-slf4j-test` which are `slf4j` implementations designed to be used for testing logging behaviors.
Then all the tests that do need to verify logging can be placed there.

Reply:
I would choose 1, but could you help to fix that ? I'm not really familiar with Gradle, sorry for that ~